SwrveSDKCommon/src/main/java/com/swrve/sdk/ISwrveCommon.java

```diff
@@ -0,0 +1,52 @@
+package com.swrve.sdk;
+
+import android.content.Context;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Map;
+
+public interface ISwrveCommon
+{
+    String getApiKey();
+
+    String getSessionKey();
+
+    short getDeviceId();
+
+    int getAppId();
+
+    String getUserId();
+
+    String getAppVersion();
+
+    String getUniqueKey();
+
+    String getBatchURL();
+
+    String getCachedLocationData();
```

Comment:
Until at least we get this refactor in. Tests are failing but soon I'll make a PR into your https://github.com/Swrve/swrve-android-sdk/pull/130 branch